docs/guides/mobile-plugin-gma-android.md

```diff
@@ -0,0 +1,58 @@
+# UID2 GMA Plugin for Android Integration Guide
+
+## Introduction
+
+The UID2 Google Mobile Ads (GMA) Plugin for Android enables publishers that use the [Google Mobile Ads (GMA) SDK](https://developers.google.com/ad-manager/mobile-ads-sdk) to include UID2 data in ad requests from Android apps. This is done automatically, with no direct coding needed from the app developer.
+
+## Functionality
+
+This plugin simplifies integration with Google Mobile Ads (GMA) for any publishers who want to support UID2 for apps running on Android devices.
+
+## Version
+
+<!-- As of 2023-07-15 -->
+
+This documentation is for the UID2 GMA Plugin for Android version 0.4.0.
+
+## GitHub Repository
+
+This plugin is in the following open-source GitHub repository:
+
+- [https://github.com/IABTechLab/uid2-android-sdk/tree/main/securesignals-gma](https://github.com/IABTechLab/uid2-android-sdk/tree/main/securesignals-gma)
+
+## Requirements 
+
+To run this plugin, install the following:
+
+1. [Google Mobile Ads SDK v22.0.0+](https://developers.google.com/admob/android/sdk)
+1. [UID2 SDK for Android v0.4.0](../sdks/uid2-sdk-ref-android.md)
+1. UID2 Android GMA Plugin v0.4.0
```

Comment:
Maybe we should add a link to where the plugin binary is published in the public maven repo 
https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk-gma

sorry this reminds me maybe we should add similar link on the UID2 SDK For Android page with:
https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk

could you add it in this PR please?

these are applicable to android only - iOS we don't publish binaries

Reply:
@sunnywu done in both files but a query re the Android SDK doc. We already list our GitHub repo as the source for the SDK. Is this an alternative source? Please clarify. Thx.

Reply:
This link here is where we use the source code to build binaries for publishers to download and use in their app. so it's different to the source code repo.